makeflow/src/makeflow.c

```diff
@@ -1420,6 +1428,9 @@ int main(int argc, char *argv[])
 		{"docker-opt", required_argument, 0, LONG_OPT_DOCKER_OPT},
 		{"amazon-config", required_argument, 0, LONG_OPT_AMAZON_CONFIG},
 		{"lambda-config", required_argument, 0, LONG_OPT_LAMBDA_CONFIG},
+		{"amazon-ami", required_argument, 0, LONG_OPT_AMAZON_AMI},
```

Comment:
AMAZON_AMI is added here and in long_opts but I don't see it handled in the cases you added. Is this a preexisting case or should it be removed.